app/src/repositories/webtoon/detailRepository.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+const db = require("../../config/db");
+
+class DetailRepository {
+  // 웹툰 상세 정보 불러오기
+  async getWebtoonByWebtoonId(webtoonId) {
+    const query = `
+      SELECT 
+        id, 
+        title, 
+        writer, 
+        illustrator, 
+        day_of_week, 
+        age_rating, 
+        description, 
+        thumbnail_url, 
+        favorite_count
+      FROM webtoons
+      WHERE id = ?
+      `;
+    try {
+      const [rows] = await db.query(query, [webtoonId]);
+      return rows[0];
+    } catch (error) {
+      console.error("DB Error [getWebtoonById]:", error);
```

Comment:
메서드명을 변경하거나, 에러의 메서드명을 실제 메서드명과 동일하게 만들거나 통일되면 좋을거같습니다.

메서드명에 대한 오탈자나 변경에 대해 신경쓰고 싶지 않으시다면 아래같은 방법도 있어요

```suggestion
      console.error(`DB Error [${DetailRepository.prototype.getWebtoonByWebtoonId.name}]:`, error); // 문법이 맞는지는 모르겠네요
```

Reply:
말씀 감사합니다! 해당 부분은 통일하지 않았던 것 같습니다. 앞으로는 메서드명과 로그를 일치시키는 방식으로 정리해두겠습니다.